internal/app/connectconformance/testsuites/cancellation.yaml

```diff
@@ -304,11 +304,11 @@ testCases:
     method: BidiStream
     streamType: STREAM_TYPE_HALF_DUPLEX_BIDI_STREAM
     cancel:
-      afterCloseSendMs: 0
+      afterCloseSendMs: 5
     requestMessages:
     - "@type": type.googleapis.com/connectrpc.conformance.v1.BidiStreamRequest
       responseDefinition:
-        responseDelayMs: 250
+        responseDelayMs: 200
```

Comment:
I could have left this at 250, since the margin here was fine. But I changed it to 200 just for consistency with the other cases.